specification/trace/semantic_conventions/messaging.md

```diff
@@ -217,6 +218,23 @@ For Apache Kafka producers, [`peer.service`](./span-general.md#general-remote-se
 The `service.name` of a Consumer's Resource SHOULD match the `peer.service` of the Producer, when the message is directly passed to another service.
 If an intermediary broker is present, `service.name` and `peer.service` will not be the same.
 
+#### CloudEvents
+
+For [CloudEvents](https://cloudevents.io/) clients, the following attributes are defined:
```

Comment:
```suggestion
For [CloudEvents](https://cloudevents.io/) clients, `messaging.system` MUST be set to `cloudevents`.
Also, the following specific attributes are defined:
```
or similar

Reply:
I added some instructions and suggestions for some of the generic attributes.